.github/workflows/training-model-mgmt-unittests.yaml

```
@@ -0,0 +1,67 @@
+name: training-model-mgmt-unittests
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - assets/training/model_management/**
+      - assets/common/components/register_model/**
+      - assets/common/src/model_registration/**
+      - assets/common/components/mlflow_model_local_validation/**
+      - assets/common/src/mlflow_model_local_validation/**
+      - .github/workflows/training-model-mgmt-unittests.yaml
+  workflow_dispatch:
+    inputs:
+      asset_dirs:
+        description: Asset directories
+        default: assets
+        required: true
```

It doesn't look like you need the `asset_dirs` input:
```suggestion
  workflow_dispatch:
```